src/app/Interface/products.ts

```diff
@@ -2,6 +2,10 @@ export interface IProduct {
   id: number,
   title: string,
   teaser: string,
-  icon: string
+  icon: string,
+  category: string,
+  image: string,
+  status: string,
+  releaseDate: any;
```

Comment:
What's the exact type for "releaseDate" might it a be a string?

Reply:
yes, right! I've changed it to string